### PR TITLE
MOSIP-26339, MOSIP-26475, MOSIP-26518, MOSIP-26538, MOSIP-26539, MOSIP-26576 : Added Test Cases for Increasing the code coverage

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

File: admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/MasterDataServiceTest.java [hunk 1]
```diff
@@ -3128,4 +3082,91 @@ public void getImmediateChildrenByLocCodeTestDataExceptionTest() {
...
+	}
+
+	@Test
+	public void buildZoneFilter_withZoneCode_returnSearchFilters() {
+		String zoneCode = "zone";
+		SearchFilter filter = new SearchFilter();
+		filter.setColumnName(MasterDataConstant.ZONE_CODE);
+		filter.setType(FilterTypeEnum.EQUALS.name());
+		filter.setValue(zoneCode);
+		ReflectionTestUtils.invokeMethod(registrationCenterService, "buildZoneFilter", zoneCode);
```
Comment (dhanendra06) [inline admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/MasterDataServiceTest.java:3112]: There is no assertion nor exception that is caught in the test method.

Comment (GOKULRAJ136) [reply admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/MasterDataServiceTest.java:3112]: Added assertions

File: admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java [hunk 2]
```diff
@@ -270,4 +285,291 @@ public void searchStartswithWildcardMasterdata04() {
 		SearchDto searchDto = new SearchDto(Arrays.asList(wildCardFilter7), Arrays.asList(sort), page, "eng");
 		searchHelper.searchMasterdata(RegistrationCenter.class, searchDto, null);	
 	}
+
+	@Test
+	public void searchMasterdataWithoutLangCode_withValidFilterAndSort_returnSearchDtoWithoutLangCode() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
```
Comment (dhanendra06) [inline admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java:292]: assertion is missing, please check for all the test cases.

Comment (GOKULRAJ136) [reply admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java:292]: Added assertions for missed test cases

**Comment by GOKULRAJ136** [comment]
The changes are merged along with this PR https://github.com/mosip/admin-services/pull/1015, hence closing this one

**Pull request closed without merging**